research/conditional/train/configs/baselines/common.yaml

```diff
@@ -0,0 +1,27 @@
+time: "2-00:00:00"
+n_gpus: 1
+interactive_debug: false
+
+# train params
+params:
+  # base bass
+  mixed_precision: true
+
+  # logging etc.
+  use_neptune: true
+  project_name: "pmtest/llm-random"
+  name: "f-bench"
+  logging_interval_heavy: 5000
+  logging_interval_loss: 1000
+  save_weights_path: "weights" # <- do sprawdzenia
+  save_weights_interval: 1000 # <- do sprawdzenia (zapisywać na koniec też)
+
+  # shallower bass
+  cutoff: 256
+  batch_size: 256
+  n_steps: 275_000
+  lr_warmup_steps: 12_500 # <- rewrite scheduling
+  lr_decay_interval: 1 # <- rewrite scheduling
+  lr_decay: 0.9999777 # <- rewrite scheduling
```

Comment:
I think the best way to implement scheduling would be to say the final learning rate and when it should happen. Like: 1/100 the original lr, in 200K steps

Reply:
Yup, a different PR will fix that. It will be merged before this one. https://trello.com/c/Egpuus34/261-fix-scheduling-remember-to-add-cosine-and-constant-at-the-end

Reply:
roger that.